kratos.gid/apps/DEM/xml/Processes.xml

```diff
@@ -199,12 +199,28 @@
           <parameter n="AngularPeriodic"  parent="LinearPeriodic" pn="Periodic" type="bool"  v="false"  help="If activated, the imposed velocity values are the maximum during a harmonic oscillation with a given period">
-           <parameter n="AngularPeriod"    parent="true"           pn="Period (s)"            v="5.0"    help="in seconds"/></parameter>
+           <parameter n="AngularPeriod"    parent="true"           pn="Period (s)"            v="5.0"    help="in seconds"/> </parameter>
           <parameter n="AngularStartTime" parent="LinearPeriodic" pn="Start time (s)"        v="0.0"    help="in seconds"/>
           <parameter n="AngularEndTime"   parent="LinearPeriodic" pn="End time (s)"          v="1000.0" help="in seconds"/>
 
+          <parameter n="Ax"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Velocity X"         values="No,Constant" pvalues="No,Constant" help="Fix Velocity X">
+           <parameter n="Vx"             parent="Constant"                    v="1.0"   pn="   Value (m/s)"                   help="X Velocity"/></parameter>
+          <parameter n="Ay"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Velocity Y"         values="No,Constant" pvalues="No,Constant" help="Fix Velocity Y">
+           <parameter n="Vy"             parent="Constant"                    v="0.0"   pn="   Value (m/s)"                   help="Y Velocity"/></parameter>
+          <parameter n="Az"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Velocity Z"         values="No,Constant" pvalues="No,Constant" help="Fix Velocity Z">
+           <parameter n="Vz"             parent="Constant"                    v="0.0"   pn="   Value (m/s)"                   help="Z Velocity"/></parameter>
+          <parameter n="Bx"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Angular Velocity X" values="No,Constant" pvalues="No,Constant" help="Fix Angular Velocity X">
+           <parameter n="AVx"            parent="Constant"                    v="0.0"   pn="   Value (rad/s)"                 help="X Angular Velocity"/></parameter>
+          <parameter n="By"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Angular Velocity Y" values="No,Constant" pvalues="No,Constant" help="Fix Angular Velocity Y">
+           <parameter n="AVy"            parent="Constant"                    v="0.0"   pn="   Value (rad/s)"                 help="Y Angular Velocity"/></parameter>
+          <parameter n="Bz"             parent="FixedDOFs" type="combo"      v="No"    pn="Fix Angular Velocity Z" values="No,Constant" pvalues="No,Constant" help="Fix Angular Velocity Z">
+           <parameter n="AVz"            parent="Constant"                    v="0.0"   pn="   Value (rad/s)"                 help="Z Angular Velocity"/></parameter>
+          <parameter n="VStart"         parent="FixedDOFs"                   v="0.0"   pn="Start time (s)"                   help="Start Time"/>
```

Comment:
Start time and end time should be an interval. See the usage in Structural App

Reply:
Checking Structural app right now.
Is there a way to change the Interval assigned to the condition in the interface?:

<ConditionItem n="SelfWeight2D" pn="Self weight" Interval="Total"

How do I change the Interval in a DEM-FEM Wall ConditionItem from Total to Custom in the interface?

thx  @jginternational

Reply:
You can change it in the last parameter. You can select any Interval from the interval section

Reply:
I see. Right now our Start and End times for the conditions are written in the mdpa. Does Interval support writing in the mdpas?

Reply:
Right, this could work